src/ai/index.js

```diff
@@ -233,132 +94,68 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
         dialogueContainer = document.getElementById('ai-chat-tool-dialogue-container');
 
         map = realityEditor.ai.mapping;
+        map.setupEventListeners();
         
         scrollToBottom();
         initTextAreaSize();
         adjustTextAreaSize();
         setupEventListeners();
-        setupSystemEventListeners();
+        // setupSystemEventListeners();
         hideEndpointApiKeyAndShowSearchTextArea();
+
+        // This is where the magic happens
+        chatInterface = new ChatInterface();
+
+        // Note: temporarily exposed as global variable for experimental purposes.
+        // If possible, minimize the use of directly accessing this outside of the ai modules.
+        realityEditor.ai.chatInterface = chatInterface;
```

Comment:
![image](https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/assets/55107443/3b27975f-b76f-44f0-ae55-18b42326bcad)

Reply:
Oh wait nevermind, I'll use realityEditor.ai.updateSummarizedState as designed. Maybe I can read

Reply:
Haha yeah actually I ended up using realityEditor.ai.chatInterface myself for convenience in one or two other spots, though I intend to clean those up. But if you find something interesting in the direct variable access that you want to make use of, feel free for now!